components/dashboard/sections/transactions-imports/SynPlaidAccountButton.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useRef } from 'react';
+import { gql, useMutation } from '@apollo/client';
+import { RefreshCw } from 'lucide-react';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import { i18nGraphqlException } from '../../../../lib/errors';
+import { API_V2_CONTEXT } from '../../../../lib/graphql/helpers';
+import { usePrevious } from '../../../../lib/hooks/usePrevious';
+
+import { Button } from '../../../ui/Button';
+import { useToast } from '../../../ui/useToast';
+
+const syncPlaidAccountMutation = gql`
+  mutation SyncPlaidAccount($connectedAccount: ConnectedAccountReferenceInput!) {
+    syncPlaidAccount(connectedAccount: $connectedAccount) {
+      id
+    }
+  }
+`;
+
+export const SynPlaidAccountButton = ({ hasRequestedSync, setHasRequestedSync, connectedAccountId, isSyncing }) => {
```

Comment:
Should we rename it to `SyncPlaidAccountButton`?

Reply:
Good catch!